.woodpecker/test.yaml

```diff
@@ -104,8 +106,8 @@ steps:
       - vendor
     image: *golang_image
     environment:
-      - WOODPECKER_DATABASE_DRIVER=postgres
-      - WOODPECKER_DATABASE_DATASOURCE=host=sv-postgres user=postgres dbname=postgres sslmode=disable
+      WOODPECKER_DATABASE_DRIVER: postgres
+      WOODPECKER_DATABASE_DATASOURCE: 'host=postgres user=postgres dbname=postgres sslmode=disable'
```

Comment:
This reverts the required hostname change related to the service names. Same for mysql.